ktor-server/ktor-server-netty/jvm/src/io/ktor/server/netty/NettyApplicationEngine.kt

```diff
@@ -75,10 +76,36 @@ public class NettyApplicationEngine(
          */
         public var tcpKeepAlive: Boolean = false
 
+        /**
+         * The maximum length of the initial line (e.g. "GET / HTTP/1.0" or "HTTP/1.0 200
+         * OK") If the length of the initial line exceeds this value, a Netty's [TooLongFrameException] will
+         * be raised.
+         */
+        public var maxInitialLineLength: Int = HttpObjectDecoder.DEFAULT_MAX_INITIAL_LINE_LENGTH
```

Comment:
I'm not sure that we should have these fields in the config, but we can keep configuration properties.
So it will be possible to configure `codec` from the `application.conf`, but keep public API simple